front/app/containers/Admin/projects/project/analysis/Insights/SummarizeButton.tsx

```diff
@@ -54,54 +64,40 @@ const SummarizeButton = () => {
     );
   }, [analysisId, filters, addSummaryPreCheck]);
 
-  const summaryPossible = !preCheck?.data.attributes.impossible_reason;
-  const summaryAccuracy = preCheck?.data.attributes.accuracy;
+  const tooManyInputs = preCheck?.data.attributes.impossible_reason;
+
+  const applyInputsLimit = !largeSummariesEnabled && inputsCount > 30;
+
+  const summaryPossible =
+    !tooManyInputs && !applyInputsLimit && inputsCount > 0;
+
+  const tooltipContent = applyInputsLimit
```

Comment:
If all inputs get filtered out, it seems that the too many inputs message will be shown?

Reply:
Nice catch! 🚀